src/compass/utils/age.py

```diff
@@ -368,7 +368,7 @@ def get_xy_info(cslc_file, mission_id='S1', pol='VV'):
         CSLC-S1 spacing along Y-direction
     '''
     if mission_id == 'S1':
-        cslc_path = '/science/SENTINEL1/CSLC/grids/'
+        cslc_path = 'CSLC/data/'
```

Comment:
```suggestion
        cslc_path = DATA_PATH
```
Same rationale as above